agenta-backend/agenta_backend/models/db_models.py

```diff
@@ -60,7 +60,6 @@ class ImageDB(Model):
     template_uri: Optional[str]
     docker_id: Optional[str] = Field(index=True)
     tags: Optional[str]
-    deletable: bool = Field(default=True)
```

Comment:
can you please explain the logic behind removing this? I remember we used it so that we never delete template images

Reply:
That's really strange. In the cloud version, 'deletable' was repeated, so I removed one. The commit: [Refactor - removed repeated 'deletable' field](https://github.com/Agenta-AI/agenta/pull/928/commits/3df53d984050d2086a301c7636debedd98ad1574). I'll add it back. Thank you for the review!